File: testdata/remove/expected/file.rb
#!/usr/bin/env ruby
# Hello World in Ruby

puts "Hello, World!"

# Define a class
class Greeter
  def initialize(name)
    @name = name
  end

  def greet
    puts "Hello, #{@name}!"
  end
end

# Create an instance and call method
greeter = Greeter.new("Ruby")
greeter.greet

File: testdata/remove/expected/file.cpp
#include <iostream>
#include <string>

class Greeter {
private:
    std::string message;
public:
    Greeter(const std::string& msg) : message(msg) {}
    void greet() {
        std::cout << message << std::endl;
    }
};

int main() {
    Greeter greeter("Hello, World!");
    greeter.greet();
    return 0;
}